AztecTests/Converters/HTMLNodeToNSAttributedStringTests.swift

```diff
@@ -24,32 +24,80 @@ class HTMLNodeToNSAttributedStringTests: XCTestCase {
         let headerNode = ElementNode(type: .h1, attributes: [], children: [spanNode1])
         let rootNode = RootNode(children: [headerNode])
 
-        // Convert + Test
+        // Convert
         let output = attributedString(from: rootNode)
 
+        // Test
         var range = NSRange()
         guard let unsupportedHTML = output.attribute(UnsupportedHTMLAttributeName, at: 0, effectiveRange: &range) as? UnsupportedHTML else {
             XCTFail()
             return
         }
 
+        let representations = unsupportedHTML.representations
         XCTAssert(range.length == textNode.length())
-        XCTAssert(unsupportedHTML.elements.count == 2)
+        XCTAssert(representations.count == 2)
 
-        let restoredSpanElement2 = unsupportedHTML.elements.last
+        let restoredSpanElement2 = representations.last
         XCTAssertEqual(restoredSpanElement2?.name, "span")
 
         let restoredSpanAttribute2 = restoredSpanElement2?.attributes.first
         XCTAssertEqual(restoredSpanAttribute2?.name, "class")
         XCTAssertEqual(restoredSpanAttribute2?.value.toString(), "aztec")
 
-        let restoredSpanElement1 = unsupportedHTML.elements.first
+        let restoredSpanElement1 = representations.first
         XCTAssertEqual(restoredSpanElement1?.name, "span")
 
         let restoredSpanAttribute1 = restoredSpanElement1?.attributes.first
         XCTAssertEqual(restoredSpanAttribute1?.name, "class")
         XCTAssertEqual(restoredSpanAttribute1?.value.toString(), "first")
     }
+
+
+    /// Verifies that BR elements contained within div tags do not cause Data Loss.
+    /// Ref. #658
+    ///
+    func testLineBreakTagWithinUnsupportedHTMLDoesNotCauseDataLoss() {
+        let inHtml = "<div><br>Aztec, don't forget me!</div>"
+        let expectedHtml = "<div><br></div><div>Aztec, don't forget me!</div>"
```

Comment:
The best scenario would be `<div><br>Aztec, don't forget me!</div>`.
We could eventually accept `<div></div><div>Aztec, don't forget me!</div>`

But this is not a valid result as it duplicates the amount of newlines.